lib/naver/searchad/api/errors.rb

```diff
@@ -49,7 +49,9 @@ class MissingRequiredAttributeError < RequestError; end
 
       class InvalidRequestError < RequestError; end
 
-      class CampaignAlreadyExistError < RequestError; end
+      class DuplicatedCampaignNameError < RequestError; end
+
+      class DuplicatedAdgroupNameError < RequestError; end
```

Comment:
What about the idiom `DuplicatedAdgroupNameError = Class.new(RequestError)`?
I like both but for an empty class that looks to suit better, what do you think?